lib/sdk/lib/ros_sdk/sdk.rb

```diff
@@ -108,9 +108,6 @@ def endpoint
         # rubocop:enable Metrics/AbcSize
       end
     end
-
-    class AppPolicy < Ros::ApplicationPolicy
```

Comment:
I think there is an include_relative somewhere that is including `Ros::ApplicationPolicy` from core. that surely needs to be removed also

Reply:
Nice catch! 👍 
Removed